src/main.h

```diff
@@ -75,13 +75,13 @@ static const CAmount HIGH_MAX_TX_FEE = 100 * HIGH_TX_FEE_PER_KB;
  */
 static const unsigned int DEFAULT_MAX_ORPHAN_TRANSACTIONS = 1000000;
 /** Default for -limitancestorcount, max number of in-mempool ancestors */
-static const unsigned int DEFAULT_ANCESTOR_LIMIT = 25;
+static const unsigned int DEFAULT_ANCESTOR_LIMIT = 500;
```

Comment:
I think we need to keep these at the BCH default, and then in a separate PR we need to implement XVERSION communication of the setting and preemptive forwarding of tx that would have been rejected from other nodes.

Reply:
reverted the commit